Northstar.CustomServers/mod.json

```diff
@@ -4,15 +4,6 @@
 	"Version": "1.19.0",
 	"LoadPriority": 0,
 	"ConVars": [
-		{
-			"Name": "ns_lobby_type",
-			"DefaultValue": "0"
-		},
-		{
-			"Name": "ns_is_modded_server",
-			"DefaultValue": "1",
-			"Flags": 8192
-		},
```

Comment:
Just to be clear, as these convars are not replicated, there shouldn't be any compatibility issues with past versions when merging and releasing this as a patch release, right? ^^"